### PR TITLE
NFT review

### PR DESCRIPTION
**Pull request by nicolaslara**

### CODE REVIEW

**Review by mattverse — COMMENTED** [review]
Thanks for the review! Don't think we should merge this and put in comments, but still found the review very useful! Leaving this PR open so that @iboss-ptk can check this as well

**Comment by mattverse** [comment]
cref: https://github.com/icns-xyz/icns/commit/9b2a6ab818d758e6a1ca19c60e0be707226da3d1

File: contracts/icns-name-nft/src/lib.rs [hunk 1]
```diff
@@ -68,6 +70,8 @@ pub mod entry {
 
         name_nft.contract_info.save(deps.storage, &info)?;
 
+        // NL: Can we set the minter on instantiation? or does this have to be a two step process?
```
[the remark this answers is not in the document]
Comment (nicolaslara) [reply contracts/icns-name-nft/src/lib.rs:73]: ahh, makes sense! (though annoying). We can probably have a contract to orchestrate this if it was gonna be deployed many times, but no point in doing that here

**Comment by nicolaslara** [comment]
thanks for the comments! The commit looks good. Closing this.

**Pull request closed without merging**